platform/AppStoreConnect/Sources/AppStoreConnect/AppStoreConnect+PrivateKeyDecoding.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+enum KeyDecodingError: Error {
+    case invalidBase64Key
+}
+
+extension AppStoreConnect {
+    
+    /// Decodes the given base64 key to AppStoreConnect-Swift-SDK suitable key format.
+    /// - Parameter key: Base64 format key to be decoded
+    /// - Returns: Decoded key suitable to be used with AppStoreConnect-Swift-SDK
+    public static func decodeBase64Key(_ key: String) throws -> String {
```

Comment:
Why would we pass a BASE64 encoded key here? It's just ASCII, right? Then we should just pass it as such. If we need to store it as BASE64 in GitHub Secrets, let the Makefile deal with that stuff.

(It wouldn't have to be stored that way, to be clear, we just do it :D)

Reply:
Well because the swift command line tool reads directly from the environment. Decoding in Makefile and then forwarding to swift CLI, I guess, would require sending it as an argument.

Reply:
Oh, didn't notice, that's a bit nasty. I think it'd be cleaner to pass the key as a file rather than to hard code some environment variable (not just name, but also encoding) here.

Reply:
Yeah, but personally I feel like we should try to contain anything that's specific to our setup with GitHub Actions, I wouldn't want to touch a dozen files in four different languages if (or if I get my way, **when**) we move away from GA... The Makefile seems like a nicer place to contain it. AFAIC you can just save it to another env variable and do the base64 decoding there, but I think a file passed to the CLI would be cleaner.

Reply:
Made changes to send the GH secrets as command arguments instead of env variables.